test/screenviewing/deltarenderer/DefaultScreenViewingDeltaRenderer.test.ts

```diff
@@ -434,4 +434,21 @@ describe('DefaultScreenViewingDeltaRenderer', () => {
       deltaRenderer.zoomReset();
     });
   });
+
+  describe('hideViewport and revealViewport', () => {
+    it('hides and reveals the viewport', () => {
```

Comment:
This test does not verify the change made by `hideViewport` or `revealViewport`. It looks like other tests in this file check properties of the canvas. Could we do the same here?